src/Utilities/model.jl

```diff
@@ -410,14 +410,31 @@ _mod(m::Module, s::Symbol) = Expr(:., m, :($(QuoteNode(s))))
 # (:Zeros) -> :(MOI.Zeros)
 _moi(s::Symbol) = _mod(MOI, s)
```

Comment:
Why not simply removing this function and force the user to write
```julia
@model LPModel () (MOI.EqualTo, MOI.GreaterThan, MOI.LessThan, MOI.Interval) (MOI.Zeros, MOI.Nonnegatives, MOI.Nonpositives) () (MOI.SingleVariable,) (MOI.ScalarAffineFunction,) (MOI.VectorOfVariables,) (MOI.VectorAffineFunction,)
```

Reply:
If you remove this _moi function and all calls to it then it should work

Reply:
FWIW the escaping/scoping rules for macros changed between 0.6 and 0.7. You might run into this.

Reply:
Forcing the module prefix sounds like the most robust way to go.